scripts/common.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash -e
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# "---------------------------------------------------------"
+# "-                                                       -"
+# "-  Common commands for all scripts                      -"
+# "-                                                       -"
+# "---------------------------------------------------------"
+
+APP_NAME="tracing-demo"
+CLUSTER_NAME="tracing-demo-space"
+GKE_VERSION=$(gcloud container get-server-config \
+  --format="value(validMasterVersions[0])")
+
+# gcloud and kubectl are required for this POC
+command -v gcloud >/dev/null 2>&1 || { \
+ echo >&2 "I require gcloud but it's not installed.  Aborting."; exit 1; }
+
+command -v kubectl >/dev/null 2>&1 || { \
+ echo >&2 "I require kubectl but it's not installed.  Aborting."; exit 1; }
+
+usage() { echo "Usage: $0 [-c <cluster name>]" 1>&2; exit 1; }
+
+# parse -c flag for the CLUSTER_NAME using getopts
+while getopts ":c:" opt; do
```

Comment:
Could you elaborate on what happens when you source this section in create, validate and teardown?

Reply:
I'm looking into removing this. If I can query it from Terraform I will.

Reply:
This has been removed.